openmrs/apps/reports/reports.json

```diff
@@ -115,7 +115,7 @@
             ],
             "showVisitInfo": true,
             "showProvider": true,
-            "formNamesToFilter": [ "Test Form"],
+            "formNamesToFilter": [ "Test Form", "SNOMED Form", "SNOMED Obs Form"],
```

Comment:
Can we give a more realistic name. What FORM is this? H&E, Pain Form, Malaria form, etc.  If this is a new H&E form, we can give it a name like: "H&E Form (SNOMED)", so it is clear this is a History and Examination Form, but supports SNOMED fields.

Reply:
Based on our milestone demo inputs, i have made the form as follows
[ "Test Form", "SNOMED H&E Form", "SNOMED Malaria Form", "SNOMED Asthma Form", "SNOMED COVID Form", "SNOMED Accident Intake Form"]